playbooks/sync-task.yml

```diff
@@ -0,0 +1,25 @@
+---
+- name: Configure instance(s)
+  hosts: all
+  become: True
+  gather_facts: No
+  tasks:
+    - name: Add script for syncing logs on exit
+      template:
+        src: sync-logs-on-exit.j2
+        dest: "{{ COMMON_BIN_DIR }}/sync-logs-on-exit"
+        owner: root
+        group: root
+        mode: 0744
+      when: COMMON_OBJECT_STORE_LOG_SYNC
+
+    - name: Force logrotate on supervisor stop
```

Comment:
@Metfriet, this is the wrong task. All the servers we are going to live patch run newer versions of Ubuntu which will not satisfy this condition. You will have to use the task immediately after that which tweaks the syslog systemd service configuration.